frontend/src/components/common/TripItemMarker/TripItemMarker.tsx

```diff
@@ -35,6 +38,13 @@ const TripItemMarker = ({ map, id, lat, lng, isSelected }: TripItemMarkerProps)
     }
   }, [id, isSelected, marker]);
 
+  useEffect(() => {
```

Comment:
```suggestion
useEffect(() => {
    const marker = new google.maps.marker.AdvancedMarkerElement({
      position: { lat, lng },
      map,
    });
    
     marker?.addListener('click', () =>  setClickedMarkerId(id));

    setMarker(marker);

    return () => {
      marker.map = null;
    };
  }, [id, lat, lng, map]);
```

굳이 새로운 useEffect를 하나 더 만들 필요 없을 듯요?!